src/ts/core/features/vim-mode/commands/navigation-commands.ts

```diff
@@ -15,6 +15,8 @@ const selectLastBlock = () => {
 export const NavigationCommands = [
     nvmap('k', 'Select Block Up', () => RoamVim.jumpBlocksInFocusedPanel(-1)),
     nvmap('j', 'Select Block Down', () => RoamVim.jumpBlocksInFocusedPanel(1)),
+    nmap('shift+h', 'Select First Visible Block', () => RoamPanel.selected().selectFirstVisibleBlock()),
+    nmap('shift+l', 'Select Last Visible Block', () => RoamPanel.selected().selectLastVisibleBlock()),
```

Comment:
What is the intuition for the default mappings? :)  
I would've though about something like `g j` `g k` (with the intuition is that `g` is for moving top/bottom + down/up mappings)

Reply:
Oh I'm basing this off of actual vim using the same keymap

Reply:
Oh, didn't know about this one 😅